controllers/pvc.go

```diff
@@ -341,3 +342,10 @@ func (r *VolumeSnapshotBackupReconciler) IsPVCBound(log logr.Logger) (bool, erro
 	return true, nil
 
 }
+
+func getDataMoverDummyPodImage() string {
+	if os.Getenv("DATA_MOVER_DUMMY_POD_IMAGE") == "" {
```

Comment:
I think this should be a RELATED_IMAGE
```suggestion
	if os.Getenv("RELATED_IMAGE_DATA_MOVER_DUMMY_POD_IMAGE") == "" {
```

Reply:
It doesn't need the `RELATED_IMAGE` prefix here. It needs it on the operator so it can be replaced with the downstream image/SHA in the operator CSV.

https://github.com/openshift/oadp-operator/pull/1053/files#diff-069ea2f824de6770c7836f0e18755a37580805128fe91bc747c41152efcea852R811

Reply:
I see wrong repo. Thx